src/server/mods.cpp

```diff
@@ -46,6 +47,9 @@ ServerModManager::ServerModManager(const std::string &worldpath) :
 	// Load normal mods
 	std::string worldmt = worldpath + DIR_DELIM + "world.mt";
 	addModsFromConfig(worldmt, gamespec.addon_mods_paths);
+
+	m_mod_counter = mb->addCounter(
+			"minetest_core_loaded_mods", "Number of loaded mods");
```

Comment:
Not an useful metric, the number of mods loaded is constant for the entire server lifetime.

Reply:
for current big servers maybe, what about new servers ?

Reply:
it's quite common in promethus world to have very stable values which changes not so many. Also imagine people wanted to have a federation of prometheus to have some metrics about various server usage at macro level, i think it's fine

Reply:
Still not convinced that this is useful. Maybe someone else has an opinion on it?

Reply:
@rubenwardy ?

Reply:
ok i will remove to ensure this PR is merged, i won't let this cool PR wait here just for that one :)